README.asciidoc

```diff
@@ -26,9 +26,157 @@ If you prefer to install and manage Elasticsearch yourself, you can download
 the latest version from 
 https://www.elastic.co/downloads/elasticsearch[elastic.co/downloads/elasticsearch].
 
-For more installation options, see the
-https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html[Elasticsearch installation
-documentation].
+=== Set up a local dev deployment
+
+//// 
+IMPORTANT: This content is replicated in the Elasticsearch guide. 
+If you make changes, you must also update setup/set-up-local-dev-deployment.asciidoc.
+////
+
+To try out Elasticsearch on your dev machine, we recommend using Docker
+and running both Elasticsearch and Kibana.
+Docker images are available from the https://www.docker.elastic.co[Elastic Docker registry].
+
+For other installation options, see the
+https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html[Elasticsearch installation documentation].
+
+**Install and run Elasticsearch**
+
+. Install and start https://www.docker.com/products/docker-desktop[Docker
+Desktop]. Go to **Preferences > Resources > Advanced** and set Memory to at least 4GB.
+
+. Start an Elasticsearch container:
++
+----
+docker network create elastic
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version} <1>
+docker run --name es-node1 --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -t docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+<1> Replace {version} with the version of Elasticsearch you want to run, such as 8.4.1.
++
+When you start Elasticsearch for the first time, the following security configuration 
```

Comment:
In a local setup do we want to deal with security by default? @tvernum do you have a strong opinion here? I hate use suggesting to people to circumvent the default but this just adds unneeded complexity. Also, as I mentioned in a previous discussion, the fact we default to JSON formatted output in Docker makes sorting through the logs a pain. To me the advantage of using docker for this scenario (ES + Kibana) is to be able to provide a simple Docker compose script.

Reply:
I don't think we should encourage people to dodge security by default. That said, it's probably sufficient here to just say that the password and enrollment token are generated automatically, rather than itemizing all of the config that happens.

Re Docker compose, it doesn't really simplify much in this one node case, and this aligns with the Kibana docs.